spec/integration_spec.rb

```diff
@@ -35,13 +35,13 @@ def executable(options={})
   end
 
   def ensure_folder(folder)
-    `mkdir -p #{folder}` unless File.exist?(folder)
+    FileUtils.mkpath(folder) unless File.exist?(folder)
   end
 
   def run_tests(test_folder, options={})
     ensure_folder folder
     processes = "-n #{options[:processes]||2}" unless options[:processes] == false
-    command = "cd #{folder} && #{options[:export]} #{executable(options)} #{test_folder} #{processes} #{options[:add]} 2>&1"
+    command = "cd #{folder} && #{options[:export]} ruby #{executable(options)} #{test_folder} #{processes} #{options[:add]} 2>&1"
```

Comment:
the ruby should go into the `executable` method ? ... maybe even conditionally on windows ?